src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/SettingsEditSettingsTwoFactor.tsx

```diff
@@ -0,0 +1,113 @@
+import { Box, Flex, Text } from "@artsy/palette"
+import * as React from "react"
+import { RelayRefetchProp, createRefetchContainer, graphql } from "react-relay"
+
+import { SystemContextProps, useSystemContext } from "v2/System"
+import { renderWithLoadProgress } from "v2/System/Relay/renderWithLoadProgress"
+import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import { SettingsEditSettingsTwoFactor_me } from "v2/__generated__/SettingsEditSettingsTwoFactor_me.graphql"
+import { SettingsEditSettingsTwoFactorQuery } from "v2/__generated__/SettingsEditSettingsTwoFactorQuery.graphql"
+
+import { SmsSecondFactorFragmentContainer } from "v2/Components/UserSettings/TwoFactorAuthentication/Components/SmsSecondFactor"
+import { BackupSecondFactorFragmentContainer } from "v2/Components/UserSettings/TwoFactorAuthentication/Components/BackupSecondFactor"
+import { AppSecondFactorFragmentContainer } from "v2/Components/UserSettings/TwoFactorAuthentication/Components/AppSecondFactor"
+
+export interface SettingsEditSettingsTwoFactorProps extends SystemContextProps {
+  me: SettingsEditSettingsTwoFactor_me
+  relay: RelayRefetchProp
+}
+
+export const SettingsEditSettingsTwoFactor: React.FC<SettingsEditSettingsTwoFactorProps> = props => {
+  const { me, relay } = props
+
+  const secondFactorFragments = () => {
+    if (typeof window !== "undefined") {
```

Comment:
This shouldn't be necessary and is indicative of some larger problem. Inline this and let's figure out what that is.

Reply:
This was something that @jacobherrington and I paired on to find a workaround. He had seen some similar errors associated with this whilst working within Braze. Tagging him here, cause he had some curiosity as to it.